components/arm/arm_test.go

```diff
@@ -345,6 +360,98 @@ func TestExtraOptions(t *testing.T) {
 	test.That(t, actualArm1.extra, test.ShouldResemble, map[string]interface{}{"foo": "bar"})
 }
 
+func TestOOBArm(t *testing.T) {
+	logger := golog.NewTestLogger(t)
+	cfg := config.Component{
+		Name:  arm.Subtype.String(),
+		Model: resource.NewDefaultModel("ur5e"),
+		ConvertedAttributes: &fake.AttrConfig{
+			ArmModel: "ur5e",
```

Comment:
i think the test should be refactored such that we test on all arm models

Reply:
I think this is actually a non-goal. The implementation details shouldn't be radically different for each arm type, and there are some dependencies on arm driver implementation (and controller-level logic, which we have no control over) for whether this will work in practice. I posit that proving this works for something based on Arm should be sufficient.